core/libp2p/host/host.hpp

```diff
@@ -49,11 +60,23 @@ namespace libp2p {
     virtual peer::PeerInfo getPeerInfo() const = 0;
 
     /**
-     * @brief Get listen addresses of the Host
-     * @note is not the same as Network::getListenAddresses(), but why - do not
-     * yet know
+     * @brief Get addresses we provided to listen on (added by listen).
+     */
+    virtual std::vector<multi::Multiaddress> getAddresses() const = 0;
+
+    /**
+     * @brief Get addresses that read from listen sockets.
```

Comment:
Hard to understand from docs difference between the three addresses. Please, collaborate more for people, who don't know it

Reply:
I thought I described precisely what each method returns...